custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -337,6 +348,102 @@ def handle_exception(self, exception, repeat_record):
                                 "followup_nikshay_registered", "followup_nikshay_error")
 
 
+@RegisterGenerator(NikshayRegisterPrivatePatientRepeater, 'case_xml', 'XML', is_default=True)
```

Comment:
Yeah, luckily, I think those names are all arbitrary, so you can just change it to, say,

```python
@RegisterGenerator(NikshayRegisterPrivatePatientRepeater, 'episode_soap_data', 'SOAP', is_default=True)
```

and I think that'll be a less surprising name, and it should work just fine.